db/sql.h

```diff
@@ -813,13 +808,35 @@ int release_locks_on_emit_row(struct sqlthdstate *thd,
 void clearClientSideRow(struct sqlclntstate *clnt);
 void comdb2_set_tmptbl_lk(pthread_mutex_t *);
 void clone_temp_table(sqlite3 *dest, const sqlite3 *src, const char *sql,
-                      int rootpg); //, pthread_mutex_t *lk);
-void sqlengine_prepare_engine(struct sqlthdstate *, struct sqlclntstate *);
-int check_thd_gen(struct sqlthdstate *, struct sqlclntstate *);
+                      int rootpg);
+int sqlengine_prepare_engine_int(struct sqlthdstate *, struct sqlclntstate *,
+                                 int recreate);
```

Comment:
Any chance we rename this to something not _int if exposed through header? If there is a good reason to share this across modules, it should have a better prototype :)

Reply:
Agree.